scripts/teamDataScript.js

```diff
@@ -0,0 +1,617 @@
+    var teamsAlreadyStoredBool = false;
+    if (checkForNull(team_QA_pairs))
+    {
+        teamsAlreadyStoredBool = true;
+    }
+
+    for (var i = 0; i < questions.length;i++)
+    {
+        var question = questions[i];
+        if (question.trim().length == 0) return;
+        
+        //loop through already stored local object: if question has an answer already stored then put that in the input box. Else, leave it empty (to fill in)
+        var potentialAnswer = '';
+        if (teamsAlreadyStoredBool == false) //len of loc storage obj == 0, nothing stored yet
+        {
+            potentialAnswer = '';
+        } else //len of loc storage obj = 1, something stored inside, now find answer to curr question
+        {
+            for (var j = 0; j < team_QA_pairs.length; j++)
+            {
+                var pair = team_QA_pairs[j];
+                var answer = pair[question];
+                if (!checkForNull(answer))
+                {
+                    continue;
+                }
+                potentialAnswer = answer;
+            }
+        }
+
+        var inputHTML = `<input type="text" value="${potentialAnswer}" class="q-input">`; //pot answer is either '' or what is stored
+        var newTRHtml = '<tr><td>' + question + '</td><td>' + inputHTML + "</td></tr>";
+        
+        if ($('#pitQuestionTable').html().indexOf(question) < 0) //if table doesn't already have it
+        {
+            $('#pitQuestionTable').append(newTRHtml);
+        }
+    }
+}
+
+function savePitAnswers() //onclick for save pit button
+{
+    //using local storage to save pit answers, one localStorage obj per team per event
+    // { '/{season}/{event}/{team}/pit answers' : {{question1}:{answer1}, {question2}:{answer2}...} }
+    //local storage obj will hold an array of q/a object pairs
+
+    var season = getCookie('currCheckedSeason');
+    var event = getCurrEvent();
+    var team = getCurrTeamNumber();
+
+    var all_team_QA_pairs_obj = [];
+    $('#pitQuestionTable tr').each(function(){ //for each row
+        var question = $(this).find('td:first').text();
+        var answer = $(this).find('td:last').find('input').val();
+
+        var pair = {};
+        pair[question] = answer;
+        if (JSON.stringify(pair).localeCompare('{}') != 0) //first row reads {} as its reading the table header
+        {
+            all_team_QA_pairs_obj.push(pair);
+        }
+    });
+
+    var objName = pitAnswerObjName(season,event,team);
+    localStorage.setItem(objName,JSON.stringify(all_team_QA_pairs_obj));
+    
+    $('#pitTableConfirmation').text('Answers saved.');
+    showAllPitDataTable();
+}
+
+function showAllPitDataTable() //shows pit stats of all teams for the current event in one table
+{
+    var season = getCookie('currCheckedSeason');
+    var event = getCurrEvent();
+    var cname = season + ' pitQuestions';
+    var questionsArr = getCookie(cname).split('Ω');
```

Comment:
`getCookie(cname).trim('Ω').split('Ω')` will yield an array without the trailing blank question is is not really there.

Reply:
That wasn't working so I just used getCookie(cname).split('Ω').slice(0,-1). After looking it up it looks like trim() doesn't accept any parameters.

Reply:
oh, I mixed up my languages again . . . (https://www.w3schools.com/python/ref_string_strip.asp),  `slice` is a good other version :)